s4e-backend/src/main/resources/db/migration/V48__product_category_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE product_category (
```

Comment:
I'd rename the file following the migration naming convention to `V48__add_product_category_table.sql`